tests/sanitize_page.codegen.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { test, expect } from '@playwright/test';
+
+// Lightweight sanitize smoke: stub heavy script to keep run fast & deterministic.
+test('sanitize page smoke', async ({ page }) => {
+  test.setTimeout(45000);
+
+  // Stub heavy sanitize.js
+  await page.route('**/Sanitization/sanitize.js', route => {
+    route.fulfill({
+      status: 200,
+      contentType: 'text/javascript',
```

Comment:
are there other assertions we can add (say an action of a click of a button, a drop down etc. It need not be super elaborate, perhaps a few behavior tests will improve confidence a lot for specific charts)